src/db-up/azext_db_up/custom.py

```diff
@@ -209,7 +209,7 @@ def _ensure_pymssql():
         python_path = os.environ.get('PYTHONPATH', '')
         os.environ['PYTHONPATH'] = python_path + ':' + db_up_ext_path if python_path else db_up_ext_path
         cmd = [sys.executable, '-m', 'pip', 'install', '--target', db_up_ext_path,
-               'pymssql==2.2.2', '-vv', '--disable-pip-version-check', '--no-cache-dir']
+               'pymssql', '-vv', '--disable-pip-version-check', '--no-cache-dir']
```

Comment:
This this safe? 🤔

Reply:
I'm also hesitant about this. But it's annoying to bump manually each time

Reply:
We are already manually bumming version in `setup.py`, right? If we want, we can loosen the version in both places.